components/templates/src/global_fns/helpers.rs

```diff
@@ -18,8 +19,12 @@ pub fn search_for_file(
     base_path: &Path,
     path: &str,
     theme: &Option<String>,
+    output_dir: Option<&Path>,
```

Comment:
in which case do we not have an output_dir?

Reply:
I assumed it would only be needed for `get_url`. Removing the option requires changing a lot more Tera functions (like `load_data`) that don't use the `output_dir`. But if you prefer removing the option, I will do that

Reply:
I think all the functions operating on files should behave the same so it would be better to change all of them

Reply:
Fair enough! I've added this now.